### PR TITLE
Adds support for JSDoc @description

### PR DESCRIPTION
**Pull request by TheOpenDevProject**
See #46.

### CODE REVIEW

File: src/documenter.ts [hunk 1]
```diff
@@ -353,7 +353,10 @@ export class Documenter implements vs.Disposable {
     }
 
     private _emitDescriptionDeclaration(sb: utils.StringBuilder, node: ts.FunctionDeclaration){
-        sb.appendLine("@description");
+        if(vs.workspace.getConfiguration().get("docthis.alwaysIncludeDescriptionTag",true)){
```
Comment (joelday) [inline src/documenter.ts]: Please add a space after "if", the comma and the last closing paren.

File: src/documenter.ts [hunk 2]
```diff
@@ -353,7 +353,10 @@ export class Documenter implements vs.Disposable {
     }
 
     private _emitDescriptionDeclaration(sb: utils.StringBuilder, node: ts.FunctionDeclaration){
-        sb.appendLine("@description");
```
Comment (joelday) [inline src/documenter.ts]: Add a blank line here.

File: src/documenter.ts [hunk 3]
```diff
@@ -353,7 +353,10 @@ export class Documenter implements vs.Disposable {
...
+        if(vs.workspace.getConfiguration().get("docthis.alwaysIncludeDescriptionTag",true)){
+            sb.appendLine("@description");
+        }
+
```
Comment (joelday) [inline src/documenter.ts]: Remove this line.

File: package.json [hunk 4]
```diff
@@ -89,6 +89,11 @@
           "type": "boolean",
           "default": true,
           "description": "When enabled, the cursor will jump to the description location on a single comment insert."
+        },
+        "docthis.alwaysIncludeDescriptionTag": {
+          "type": "boolean",
+          "default": true,
+          "description": "When enabled, JSDoc comments will always include @description"
```
Comment (joelday) [inline package.json]: Add a period at the end.

File: src/documenter.ts [hunk 5]
```diff
@@ -361,6 +368,7 @@ export class Documenter implements vs.Disposable {
         this._emitParameters(sb, node);
         this._emitReturns(sb, node);
         this._emitMemberOf(sb, node.parent);
+        this._emitDescriptionDeclaration(sb,node);
```
Comment (joelday) [inline src/documenter.ts]: Add a space after the comma.

**Comment by TheOpenDevProject** [comment]
Can do, will commit to this PR tonight.

**Comment by TheOpenDevProject** [comment]
Not sure why this fails the CI build. Any insight would be good.

**Comment by joelday** [comment]
@TheOpenDevProject Sorry, I added tslint to the build process to help with future PRs. It looks like it hit the following errors:

src/documenter.ts[354, 95]: missing whitespace
src/documenter.ts[355, 87]: missing whitespace

Anyway, I'll take care of this. Thank you for spending the time and effort on this. It's extremely helpful to have people to participate in improving the extension.

**Pull request merged**